applications/DFG.prm

```diff
@@ -17,70 +17,70 @@ subsection Navier-Stokes solver parameters
   set Convective term time discretization    = semi-implicit
   set Convective term weak form              = skew-symmetric
   set Incremental pressure-correction scheme = rotational
-  set Preconditioner update frequency        = 15
+  set Preconditioner update frequency        = 1
```

Comment:
This modification caused the problems described below. Setting the preconditioner frequency to unity at the beginning of a simulation is not advisable...

Reply:
I am a bit confused, the commit changes all the values to reproduce the error or to fix the error? Here you comment that an update of the preconditioner each step is not advisable but that what is being committed.

Reply:
Forget this comment, it did not fix anything in the end.